pyomo/gdp/plugins/chull.py

```diff
@@ -723,6 +706,8 @@ def _xform_constraint(self, obj, disjunct, infodict, var_substitute_map,
         if obj.is_indexed():
             try:
                 newConstraint = Constraint(obj.index_set(), transBlock.lbub)
+            # ESJ: TODO: John, is this except block still reachable in the
+            # post-set-rewrite universe? I can't figure out how to test it...
```

Comment:
No.  In fact, I bet with a little work, we could get rid of the `if obj.is_indexed()`, too...